includes/class-sensei-frontend.php

```diff
@@ -49,17 +49,7 @@ class Sensei_Frontend {
 	 */
 	public function __construct() {
 
-		$this->allowed_html = array(
-			'embed'  => array(),
-			'iframe' => array(
-				'width'           => array(),
-				'height'          => array(),
-				'src'             => array(),
-				'frameborder'     => array(),
-				'allowfullscreen' => array(),
-			),
-			'video'  => Sensei_Wp_Kses::get_video_html_tag_allowed_attributes(),
-		);
+		$this->allowed_html = Sensei_Wp_Kses::get_default_wp_kses_allowed_html();
```

Comment:
I think we should consider also allowing any tags here that `wp_kses_post` allows:
```
$this->allowed_html = array_merge( Sensei_Wp_Kses::get_default_wp_kses_allowed_html(), wp_kses_allowed_html( 'post' ) );
```

Reply:
i think you are right but this is a field for any video  so we can't allow other tags,  i thing does'e need to allow wp_kses_allowed_html( 'post' ),

Reply:
updateds now

Reply:
I disagree. For example, [Vimeo embeds](https://vimeo.com/347119375) include HTML tags like `<p>` and `<a>` that would be stripped out by this.

Besides, if we're allowing people to [save HTML in the video embed code for a lesson in WP Admin](https://github.com/Automattic/sensei/pull/2992/files), but then don't display it on the front-end, what's the point?